README.md

```diff
@@ -24,29 +24,26 @@
 ### Step 2: Configure HashiCorp Vault extension
 
 1. Build the HashiCorp Vault Integration OSGI bundle using `mvn clean install` and copy
```

Comment:
This doesn't contain information about the validity of this feature. With what products this can be used, the minimum dependency versions and so on.

Reply:
added info